### PR TITLE
Update target frameworks

### PR DESCRIPTION
**Pull request by ShivangiReja**
Update target framework: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1832893&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=45862c6e-83c5-515f-73e2-e7009eff9f9b&l=33

Resolve broken links: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1832893&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=bdeefc16-b669-5ebd-ad94-a2c19ade53b0&l=38

### CODE REVIEW

File: sdk/search/Microsoft.Azure.Search/CHANGELOG.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ None
 
 ### Breaking Changes
 
-For more details, visit the [SDK migration guide](https://docs.microsoft.com/azure/search/search-dotnet-sdk-migration-version-10).
+For more details, visit the SDK migration guide.
```
Comment (jsquire) [inline sdk/search/Microsoft.Azure.Search/CHANGELOG.md:45]: Did we intend to remove the link to the migration guide?   Seems that'd be helpful!

**Review by jsquire — APPROVED** [review]

File: sdk/search/Microsoft.Azure.Search/CHANGELOG.md (hunk 1 above)
Comment (ShivangiReja) [reply sdk/search/Microsoft.Azure.Search/CHANGELOG.md:45]: These links are broken and don't exist.

Comment (pallavit) [reply sdk/search/Microsoft.Azure.Search/CHANGELOG.md:45]: Is this the one you want - https://docs.microsoft.com/en-us/azure/search/search-dotnet-sdk-migration-version-11?

**Pull request merged**